packages/FormElement/src/FormElement.js

```diff
@@ -132,6 +132,24 @@ function FormElement(props) {
     "aria-invalid": hasError,
   };
 
+  const componentNameMap = {
+    DATEPICKER: "DatePicker",
+    INPUT: "Input",
+    LISTBOX: "ListBox",
+  };
+
+  const supportedComponents = [componentNameMap.DATEPICKER, componentNameMap.INPUT, componentNameMap.LISTBOX];
```

Comment:
I'll be honest I don't have a good alternative, but special casing components like that just seems really off to me. I guess this is a consequence of the overall API here, which again is based on inspecting your direct children and their types. 😞